### PR TITLE
Add a path rule for static routing

### PR DESCRIPTION
**Pull request by hweej**

### CODE REVIEW

File: public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml [hunk 1]
```diff
@@ -55,3 +54,46 @@ spec:
...
+    - host: cellxgene.cbioportal.org
+      http:
+        paths:
+          - path: /static(/|$)(.*)
+            pathType: Prefix
+            backend:
+              service:
+                name: cellxgene-service-1
+                port:
+                  number: 8080
```
Comment (dippindots) [inline public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml:99]: Could we add this part together with the other three paths?

Comment (dippindots) [reply public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml:99]: I think we don't need to include those annotations again.

Comment (dippindots) [reply public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml:99]: And just want to double-check with you that this one is also routing to `cellxgene-service-1`, right?

**Comment by hweej** [comment]
Unfortunately, we have to create a separate ingress for the `/static` path.  This is because we can only have **one** annotation nginx.ingress.kubernetes.io/rewrite-target per set of paths. We need a new rewrite target because /static is a special case for this set of services.

File: public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml (hunk 1 above)
Comment (hweej) [reply public-eks/cbioportal-prod/cellxgene/cellxgene-ingress.yaml:99]: yes, this is correct. Basically what I'm doing is that, I'm telling all other deployments to grab a static asset from the `cellxgene-service-1`.

**Review by dippindots — APPROVED** [review]

**Pull request merged**